pkg/v1/internal/estargz/constants.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package estargz
+
+// TOCDigestKey holds the key of the layer annotation used to associate
+// table-of-contents verification information to the layer.
+const TOCDigestKey = "containerd.io/snapshot/stargz/toc.digest"
```

Comment:
Pull this from https://github.com/containerd/stargz-snapshotter/blob/af03a142bbeefb0e07e915e4e8fdedc082e5039b/estargz/types.go#L71 instead?

Reply:
done